### DISCUSSION TITLE
I can't use the Firemote card when my firecube first turns on without first using the fire tv remote app.

### DISCUSSION THREAD

**Discussion by mizuxe** [PRProd/HA-Firemote#447] [Q&A] [answered]
Hello, I love this card and this community, and @PRProd in general <3.  I have felt a bit lost in the ADB documentation and I'm just trying to resolve a specific issue that happens when my fire cube needs to be restarted. My firemote card doesn't work when the fire tv is first turned on, unless I first connect to my tv through the fire tv remote app, and then the firemote works great until the fire cube loses power. Is there something I'm missing in my ADB setup? In the hass.io docs they mention ADB locking, but I'm not sure what's going on here.

**Comment by PRProd** [comment, marked as answer]
Awww... thanks @mizuxe!  That's very nice of you to say! ❤️ 

What you've described is a known limitation with the Amazon Fire TV line of products when using certain types of ADB commands.  The official ADB documentation doesn't / wouldn't include this nuance, which is probably why you were unable to find the answer there.  I've done my best to try to include this information in a few places here in this repo, but it's easy to miss.  This same question came up again just a few weeks ago in discussion #396 as well.

If you take a look at the [FAQ](https://github.com/PRProd/HA-Firemote?tab=readme-ov-file#faq) section in the [README.md](https://github.com/PRProd/HA-Firemote/blob/main/README.md) file, you will find [this](https://github.com/PRProd/HA-Firemote?tab=readme-ov-file#why-dont-any-of-the-buttons-on-the-firemote-work-at-all):

![image](https://github.com/PRProd/HA-Firemote/assets/25486599/35b56df1-8321-4f94-b43c-42819da4c416)

Similar references to this issue are also noted in the [Known Limitations](https://github.com/PRProd/HA-Firemote/wiki#known-limitations) section, and the [FAQ](https://github.com/PRProd/HA-Firemote/wiki#why-dont-any-of-the-buttons-on-the-firemote-work-at-all) section of the [Firemote Wiki](https://github.com/PRProd/HA-Firemote/wiki)

I wish there was a good way to solve this issue.  I've tried a lot of things but have had no success.  Feel free to [take a look at this link](https://github.com/PRProd/HA-Firemote/discussions/115#discussioncomment-5970362) to witness my pain.

The best answer I can provide is what I wrote in discussion #396, so I will copy/paste that info here as well:

> There is one way to work around this, but using this solution does introduce another behavior that you might not like. If you switch your Compatibility Mode to **Strong (Slower)** like this:
>
> ![image](https://github.com/PRProd/HA-Firemote/assets/25486599/e271877e-3409-4f4e-a1c8-7482bc1ea2b7)
>
> The disconnection problems you have will disappear, but at a cost. The navigation buttons will have quite a lag between the time you press the button on your Firemote, and when your device responds.
>
> The best advice I can provide is to either keep your device powered on, keep your physical remote handy for the "press one button" solution, or switch to the **Strong (Slower)** Compatibility Mode.

I hope this helps!

-Doug

**Reply by mizuxe** [reply]
Thank you Doug! I just wanted to make sure it wasn't something that could be easily fixed.

**Reply by PRProd** [reply]
You're very welcome, @mizuxe !
